cluster/examples/kubernetes/ceph/upgrade-from-v1.0-create.yaml

```diff
@@ -129,3 +128,301 @@ subjects:
   - kind: ServiceAccount
     name: rook-ceph-system
     namespace: ROOK_SYSTEM_NAMESPACE
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-csi-cephfs-plugin-sa
+  namespace: rook-ceph
```

Comment:
Wherever you have `namespace: rook-ceph` we will need to replace with `namespace: ROOK_NAMESPACE` or `namespace: ROOK_SYSTEM_NAMESPACE` so the user can substitute the actual namespace.